src/Users/UserFilters.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+namespace HelpScout\Api\Users;
+
+use HelpScout\Api\Assert\Assert;
+
+class UserFilters
+{
+    /**
+     * @var int
+     */
+    private $mailbox;
+
+    /**
+     * @var string
+     */
+    private $email;
+
+    public function getParams(): array
+    {
+        $params = [
+            'mailbox' => $this->mailbox,
+            'email' => $this->email,
+        ];
+
+        // Filter out null values
+        return array_filter($params, function ($param) {
```

Comment:
I was just made aware recently that `array_filter` actually filters out null items by default. 

So I _think_ this could even be: 

```
return array_filter($params);
```

> If no callback is supplied, all entries of array equal to FALSE (see converting to boolean) will be removed. -- [Documentation](https://secure.php.net/manual/en/function.array-filter.php)

Reply:
👍 Nice.  This has been updated